clr/main.py

```diff
@@ -13,13 +29,41 @@ def main(argv=None):
     namespace = get_namespace(namespace_key)
     bound_args = namespace.parse_args(cmd_name, argv[2:])
 
-    # Call the command.
+    init_beeline(namespace_key)
+    trace = beeline.start_trace(context={
+        "name": cmd_name,
+        "username": getpass.getuser(),
+        "hostname": socket.gethostname(),
+    })
+
     if hasattr(namespace.instance, "cmdinit"):
+        cmdinit_span = beeline.start_span({"name": "cmdinit"})
         namespace.instance.cmdinit()
-    result = namespace.command_callables[cmd_name](
-        *bound_args.args, **bound_args.kwargs
-    )
+        beeline.finish_span(cmdinit_span)
```

Comment:
Ahh, that would work. I had that initially, but threw it away due to some other things - those are no longer relevant so that would work. I'll change it up.

Reply:
done